scripts/get_localtime.py

```diff
@@ -0,0 +1,5 @@
+import time
+
+t = time.localtime()
+current_time = time.strftime("%I:%M %p", t)
+print("It is", current_time)
```

Comment:
Add a new line at the end of the file, it seems like github is complaining about it.